content/content-strategy/identify-business-needs/what-business-needs/mission.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: text/textblock
+---
+
+### [2]Mission
```

Comment:
Comparing all the files in this PR, there are no heading 2s above this section. This heading should therefore be a heading 2 and placed in its own `nav/section` markdown file so we ensure correct partial use.

Instead of `### [2] Mission` there should be a separate file and the value should just have 'Mission'. This new `nav/section` file will need to have the same heading value as a table of contents, which is also missing from the collection of markdown for this page. Please revise.